tools/pythonpkg/src/python_conversion.cpp

```diff
@@ -201,7 +201,7 @@ bool TryTransformPythonNumeric(Value &res, py::handle ele) {
 	return true;
 }
 
-Value TransformPythonValue(py::handle ele, const LogicalType &target_type) {
+Value TransformPythonValue(py::handle ele, const LogicalType &target_type, bool from_pandas) {
```

Comment:
So the difference here is that nan is converted to null if it comes from Pandas, right?
Isn't a  nan_as_null a bit more descriptive?

Reply:
Yes I agree 👍